docs/_articles/en/getting-started/java-setup.md

```diff
@@ -73,7 +73,7 @@ Windows:
 
 ## Oracle Java 11 (Officially Test)
 
-1. Navigate to the [download page](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
+1. Navigate to the [download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk11-downloads-5066655.html)
```

Comment:
We've seen users download the wrong jdk because this link takes them to a page with all the versions, with the top one being 13. This links directly to 11.